app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,58 @@
+<h1 class="text-center text-primary text-uppercase heading-text mb-3">Recipe <%= @recipe.id %></h1>
+<div class="d-flex flex-column gap-3">
+  <div class="d-flex align-items-baseline justify-content-between lead">
+    <div class="d-flex flex-column gap-2">
+      <p>Preparation time: <%= @recipe.preparation_time %> minutes</p>
+      <p>Cooking time: <%= @recipe.cooking_time %> minutes</p>
+      <p>Steps goes here...</p>
+    </div>
+    <div>
+      <% if can?(:manage, @recipe) %>
+        <%= form_for @recipe, url: toggle_public_recipe_path(@recipe), method: :patch, remote: true, class: 'toggle-form' do |f| %>
+          <%= f.hidden_field :authenticity_token, value: form_authenticity_token %>
+          <span>Public<span>
+          <label class="switch">
+            <%= f.check_box :public, { class: 'form-check-input', onchange: 'this.form.submit();' }, true, false %>
+            <span class="slider round"></span>
+          </label>
+        <% end %>
+      <% end %>
+    </div>
+  </div>
+
+  <div class="border-bottom border-primary border-2 pb-3">
+    <div class="d-flex justify-content-between">
+      <%= link_to 'Generate shopping list', general_shopping_list_path, class: 'btn btn-primary' %>
+      <% if can? :manage, @recipe %>
+        <%= link_to 'Add Food', new_recipe_recipe_food_path(@recipe), class: 'btn btn-primary' %>
+      <% end %>
+    </div>
+  </div>
+
+  <div class="mt-4">
+    <table class="table border w-full">
+      <thead class="border border-black table-dark">
+        <tr class="border text-2xl">
+          <th class="border p-2">Food</th>
+          <th class="border p-2">Quantity</th>
+          <th class="border p-2">Value</th>
+          <th class="border p-2">Actions</th>
+        </tr>
+      </thead>
+      <tbody class="border border-black">
+        <% @recipe_foods.each do |recipe_food| %>
+          <tr class="border border-black">
+            <td class="border  border-black p-1"><%= recipe_food.name %></td>
+            <td class="border  border-black p-1"><%= recipe_food.quantity %></td>
+            <td class="border  border-black p-1">$<%= recipe_food.price %></td>
+            <td class="border  border-black p-1">
```

Comment:
- [OPTIONAL] The essence of having a `Recipe#Show` Page is to ensure that it displays details of the recipe including the food, food price and the quantity needed to make that recipe when the user was creating the recipe using the form. However, I noticed that your application displays the details of each food item created instead of the actual details of the food required to make the recipe.

For example, if I created rice as a food item and I have **_20 grams_** of _**rice**_ at creation, then I create a recipe that uses **_40 grams_** of **_rice_**, in the `Recipes#Show` page, the details should show **_40 grams_** of _**rice**_ with the amount calculated automatically instead of the **_20grams_** of _**rice**_ because **_40 grams_** of _**rice**_ is what is needed to create the food. It would be nice to fix this. 👍 😎 


### **_Recipe#Show Page_**

![image](https://github.com/sahlminkok/recipe-app/assets/77106962/7b64a4c8-108b-425b-884b-15f613fb6dd9)

### _**Food#Index Page**_

![image](https://github.com/sahlminkok/recipe-app/assets/77106962/8f3c8751-0b2c-4959-be48-27c8a2f403dd)

### Shopping List (To show the quantity Needed)

![image](https://github.com/sahlminkok/recipe-app/assets/77106962/0538ea0b-e813-4171-b15b-970f6dea4c8d)